### DISCUSSION TITLE
Ajout de nouveaux effets (potions)

### DISCUSSION THREAD

**Discussion by QQtin3** [Crownicles/Crownicles#2155] [Nouvelles idées]
Proposition faite par 🇲🇦يونس🇫🇷 :

- Annule tous les effets négatifs pendant le prochain combat (genre si on se fait étourdire et bah ça nous protèges et donc l'étourdissement s'annule)
- rajoute + x pv durant chaque tours du combat
- A une probabilité de permettre au porteur de l'objet d'esquiver l'attaque de l'adversaire
- Permet de se déplacer instantanément sur un lieu qu'on veux, genre comme de la téléportation, ça peut être une fois par j ou ça peut être une potion
- à une probabilité de persuader l'ennemi de se faire du mal à lui même pour le prochain tour (sa prochaine attaque)
- Double l'xp/l'or qu'on gagne au prochain event/durant une durée
- Invisibilité, ferais que par exemple la sorsière nous verrais pas, le mec en dragon là, les pets etc, pour avoir que des mini events où - y'a pas besoin d'être aperçu
- Augmentation des coups critiques
- Ah oui et les potions sans effets pour que ce soit un effet mais inconnu (des trucs bien et mal)
- Genre ça peu nous faire avancer de 5 h comme nous faire perdre 50 pv
- x énergie quand on le porte

**Comment by Nargacaura** [comment]
Alors:

> Annule tous les effets négatifs pendant le prochain combat (genre si on se fait étourdire et bah ça nous protèges et donc l'étourdissement s'annule)

Ça serait cheaté si on serait entièrement protégé lors d'un combat vu que les altés sont une partie intégrante du combat (comme une alternative au côté bourrin).

> rajoute + x pv durant chaque tours du combat

Ça va pas créer une situation similaire à un lanceur spammant des repos?

> A une probabilité de permettre au porteur de l'objet d'esquiver l'attaque de l'adversaire

Les probas d'échec et de précision font déjà le taf'. Mais si un objet devrait exister, ça sera pas plus de 5-10%. Trop haut serait cheaté.

> Permet de se déplacer instantanément sur un lieu qu'on veux, genre comme de la téléportation, ça peut être une fois par j ou ça peut être une potion

Pas fan de l'idée, ça pourrait spammer les lieux où y'a le plus de points en jeu si c'est un objet.

> à une probabilité de persuader l'ennemi de se faire du mal à lui même pour le prochain tour (sa prochaine attaque)

J'aurai plus vu ça comme une alté "Confus", et non un objet.

> Double l'xp/l'or qu'on gagne au prochain event/durant une durée

Pourquoi pas, comme un sorte de jackpot?

> Invisibilité, ferais que par exemple la sorsière nous verrais pas, le mec en dragon là, les pets etc, pour avoir que des mini events où - y'a pas besoin d'être aperçu

Je pense que s'il veut tomber à 90% (ok, j'extrapole un peu) sur du doNothing, peut-être. Mais n'avoir que du doNothing n'est pas si fun que ça en a l'air...

> Augmentation des coups critiques

Pareil que pour l'esquive. 5-10% max, pas plus.

> les potions sans effets pour que ce soit un effet mais inconnu (des trucs bien et mal)

À quoi serviraient les malus, pour être franc? Ça risquerait d'être aussi inutile que des popos de combats pour quelqu'un qui ne se bat jamais...

> x énergie quand on le porte

Pas sûr qu'on puisse changer l'énergie de quelqu'un comme ça, vu que ça se base sur la classe etle niveau uniquement.

Tous ces points sont juste mon avis, btw.

**Reply by QQtin3** [reply]
Ton avis me semble cohérent, je ne fais que relayer des propos, après pour les malus bon, les missions peuvent te demander de boire une potion sans effet, donc pourquoi ne pas instaurer une part de chance aléatoire qui peut te permettre de gagner/rien/perdre à faible dose, j'avais déjà évoqué cette proposition il y a quelques temps maintenant et je trouve ça plutôt ok tant que ça reste pas abusé niveau probas

**Comment by Gabrielre14** [comment]
Personnellement j'aime beaucoup l'idée d'un résultat aléatoire pour les potions sans effet, transformée en potion à effet inconnu (ou pourquoi pas rajouter un nouveau type de potion à effet inconnu). 
Ce serais un bonus vraiment random, genre autant gagner des pièces que de la vie que de l'énergie que du temps, voir pourquoi pas un animal (je pense que les rédacteurs ou moi auront assez d'imagination pour trouver une explication cohérente à trouver un animal en buvant une potion ;) ), ou quelques malus (mais pas aussi grand que le -50 évoqué au dessus ^^')
Et je propose que les bonus augmentent avec la rareté et pourquoi pas les malus aussi 👀 (avec peut-être une baisse de proba)
Un peu une potion quitte ou double à une haute rareté

Concernant le fait d'annuler les effets négatifs pourquoi pas faire une potion spécifique à un effet. Par exemple, une potion qui protège de l'étourdissement. C'est une possibilité. Et comme elle reste cheat on peut la limiter à des potions de haute rareté
